src/main/java/com/gmail/goosius/siegewar/command/SiegeWarAdminCommand.java

```diff
@@ -173,6 +178,122 @@ private void parseSiegeWarAdminCommand(CommandSender sender, String[] args) {
+		// Populate nation ranks.
+		groupNodes = TownyPerms.getPermsOfGroup("nations.ranks.private");
+		groupNodes.add(nationpoints);
+		groupNodes.add("towny.nation.siege.pay.grade.100");
+		file.set("nations.ranks.private", groupNodes);
+
+		groupNodes = TownyPerms.getPermsOfGroup("nations.ranks.sergeant");
+		groupNodes.add(nationpoints);
+		groupNodes.add("towny.nation.siege.pay.grade.150");
+		file.set("nations.ranks.sergeant", groupNodes);
+
+		groupNodes = TownyPerms.getPermsOfGroup("nations.ranks.lieutenant");
+		groupNodes.add(nationpoints);
+		groupNodes.add("towny.nation.siege.pay.grade.200");
+		file.set("nations.ranks.lieutenant", groupNodes);
+
+		groupNodes = TownyPerms.getPermsOfGroup("nations.ranks.captain");
+		groupNodes.add(nationpoints);
+		groupNodes.add("towny.nation.siege.pay.grade.250");
+		file.set("nations.ranks.captain", groupNodes);
+
+
+		groupNodes = TownyPerms.getPermsOfGroup("nations.ranks.major");
+		groupNodes.add(nationpoints);
+		groupNodes.add("towny.nation.siege.pay.grade.300");
+		file.set("nations.ranks.major", groupNodes);
+
+		groupNodes = TownyPerms.getPermsOfGroup("nations.ranks.colonel");
+		groupNodes.add(nationpoints);
+		groupNodes.add("towny.nation.siege.pay.grade.400");
+		file.set("nations.ranks.colonel", groupNodes);
+
+		groupNodes = TownyPerms.getPermsOfGroup("nations.ranks.general");
+		groupNodes.add("siegewar.nation.siege.*");
+		groupNodes.add("towny.command.nation.rank.private");
+		groupNodes.add("towny.command.nation.rank.sergeant");
+		groupNodes.add("towny.command.nation.rank.lieutenant");
+		groupNodes.add("towny.command.nation.rank.captain");
+		groupNodes.add("towny.command.nation.rank.major");
+		groupNodes.add("towny.command.nation.rank.colonel");
+		groupNodes.add("towny.nation.siege.pay.grade.500");
+		file.set("nations.ranks.general", groupNodes);
+		
+		// Add nodes to king rank.
+		groupNodes = TownyPerms.getPermsOfGroup("nations.king");
+		if (!groupNodes.contains("siegewar.nation.siege.*"))
+			groupNodes.add("siegewar.nation.siege.*");
+		if (!groupNodes.contains("siegewar.command.siegewar.nation.*"))
+			groupNodes.add("siegewar.command.siegewar.nation.*");
+		file.set("nations.king", groupNodes);
+		
+		// Add nodes to the nation assistant rank.
+		if (TownyPerms.mapHasGroup("nations.ranks.assistant")) {
+			groupNodes = TownyPerms.getPermsOfGroup("nations.ranks.assistant");
+			if (!groupNodes.contains("siegewar.nation.siege.*"))
+				groupNodes.add("siegewar.nation.siege.*");
+			if (!groupNodes.contains("siegewar.command.siegewar.nation.*"))
+				groupNodes.add("siegewar.command.siegewar.nation.*");
+			file.set("nations.ranks.assistant", groupNodes);
+		}
+		file.save();
+		
+		// TODO: make wiki page or link to section of install guide which lists the remaining nodes for staff/cannons/capital switching.
+		Messaging.sendMsg(sender, "Permission nodes installed: see wiki for nodes you should manually give, which cannot be done automatically: http://some.url.here");

```

Comment:
:construction: 
- The link at the end of this message, which gets shown to the admin, is still a placeholder
- Fyi the install guide is: https://github.com/TownyAdvanced/SiegeWar/wiki/Siege-War-Installation  , but Im not sure how to link to a particular section in there.

Reply:
Yeah I think the guide will have to be altered at the time of release, indicating that the command is to be run, and that the further optional things should be considered.

Reply:
Ok. I added a ticket to the 0.5.0 milestone so we don't forget to do that before release.
https://github.com/TownyAdvanced/SiegeWar/issues/349